notebooks/test_mapping_threshold.ipynb

```diff
@@ -2,39 +2,41 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Database(MongoClient(host=['selected-tests-shard-00-01-cf7ik.mongodb.net:27017', 'selected-tests-shard-00-02-cf7ik.mongodb.net:27017', 'selected-tests-shard-00-00-cf7ik.mongodb.net:27017'], document_class=dict, tz_aware=False, connect=True, authsource='admin', replicaset='selected-tests-shard-0', ssl=True, retrywrites=True, w='majority'), 'selected_tests')\n",
-      "[{'$lookup': {'from': 'test_mappings_test_files', 'localField': '_id', 'foreignField': 'test_mapping_id', 'as': 'test_files'}}, {'$unwind': '$test_files'}, {'$addFields': {'threshold': {'$divide': ['$test_files.test_file_seen_count', '$source_file_seen_count']}}}, {'$group': {'_id': None, 'thresholds': {'$push': '$threshold'}}}]\n",
-      "14432\n"
+      "2867876\n"
      ]
     }
    ],
    "source": [
     "from pymongo import MongoClient\n",
+    "import os\n",
     "client = MongoClient(os.environ.get(\"SELECTED_TESTS_MONGO_URI\"))\n",
     "\n",
     "db = client.selected_tests\n",
-    "pipeline = [{\"$lookup\": {\"from\": \"test_mappings_test_files\", \"localField\": \"_id\",  \"foreignField\": \"test_mapping_id\", \"as\": \"test_files\" }}, {\"$unwind\": \"$test_files\"}, {\"$addFields\": {\"threshold\": {\"$divide\": [\"$test_files.test_file_seen_count\", \"$source_file_seen_count\"]}}}, { \"$group\" : { \"_id\" : None, \"thresholds\": {\"$push\": \"$threshold\"}}}]\n",
```

Comment:
Had to remove the $group because it was returning such a large list of thresholds that I was getting the following error:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/lydia.stepanek/.pyenv/versions/selected-tests-3.7.3/lib/python3.7/site-packages/pymongo/collection.py", line 2411, in aggregate
    **kwargs)
  File "/Users/lydia.stepanek/.pyenv/versions/selected-tests-3.7.3/lib/python3.7/site-packages/pymongo/collection.py", line 2318, in _aggregate
    user_fields={'cursor': {'firstBatch': 1}})
  File "/Users/lydia.stepanek/.pyenv/versions/selected-tests-3.7.3/lib/python3.7/site-packages/pymongo/pool.py", line 584, in command
    user_fields=user_fields)
  File "/Users/lydia.stepanek/.pyenv/versions/selected-tests-3.7.3/lib/python3.7/site-packages/pymongo/network.py", line 158, in command
    parse_write_concern_error=parse_write_concern_error)
  File "/Users/lydia.stepanek/.pyenv/versions/selected-tests-3.7.3/lib/python3.7/site-packages/pymongo/helpers.py", line 155, in _check_command_response
    raise OperationFailure(msg % errmsg, code, response)
pymongo.errors.OperationFailure: BSONObj size: 47642809 (0x2D6F8B9) is invalid. Size must be between 0 and 16793600(16MB) First element: _id: null
```